system_test/aest_sync_SUITE.erl

```diff
@@ -0,0 +1,190 @@
+-module(aest_sync_SUITE).
+
+%=== EXPORTS ===================================================================
+
+% Common Test exports
+-export([all/0]).
+-export([init_per_testcase/2]).
+-export([end_per_testcase/2]).
+
+% Test cases
+-export([new_node_joins_network/1,
+         docker_keeps_data/1,
+         crash_and_continue_sync/1]).
+
+-import(aest_nodes, [
+    setup_nodes/2,
+    start_node/2,
+    stop_node/2, stop_node/3,
+    kill_node/2,
+    http_get/5,
+    request/4,
+    wait_for_height/4,
+    assert_synchronized/2
+]).
+
+%=== INCLUDES ==================================================================
+
+-include_lib("eunit/include/eunit.hrl").
+
+%=== MACROS ====================================================================
+
+% -define(MINING_TIMEOUT, 4*10000).
+-define(MINING_TIMEOUT, 1000).
+
+-define(NODE1, #{
+    name    => node1,
+    peers   => [node2],
+    backend => aest_docker,
+    source  => {pull, "aetrnty/epoch:v0.8.0"}
+}).
+
+-define(NODE2, #{
+    name    => node2,
+    peers   => [node1],
+    backend => aest_docker,
+    source  => {pull, "aetrnty/epoch:v0.8.0"}
+}).
+
+-define(NODE3, #{
+    name    => node3,
+    peers   => [node1],
+    backend => aest_docker,
+    source  => {pull, "aetrnty/epoch:local"}
```

Comment:
Do I understand correctly this is meant to be current master hence this can be [aeternity:dev](https://hub.docker.com/r/aeternity/epoch/tags/) ? ([Current pushes happen on aeternity Docker org](https://github.com/aeternity/epoch/blob/e65c7e4d838db29b52f371ab97c96eaae84bf717/.circleci/config.yml#L13).)

I ask as I am thinking of attempting to hook this CI after this is merged.

Reply:
I would point to an image that is built locally, even on CI, so you know which exact revisions you are testing. If a locally built Docker image called `aetrnty/epoch:dev` exists locally on CI, then that's a good choice.

Reply:
I see - so recent master shall be ok. Thanks.

Reply:
I'll change the test to use `aetrnty/epoch:dev` and change the wiki to have people build the dev image from the local sources.

Reply:
For the system tests on CI we're now using `local`? So this should not be changed?

Reply:
Yes, this comment was before we ended up deciding to keep using `local`. Not changing it.